### DISCUSSION TITLE
Add Grimoires and Scripts to ChatAnnouncements

### DISCUSSION THREAD

**Discussion by saenic** [DakJaniels/LuiExtended#296] [Ideas]
I think this would be a good idea. 
Sadly I don't have time to do it. Maybe @ACastanza or @DakJaniels can do it?

**Comment by DakJaniels** [comment]
It is on my todo list. 😬

**Comment by DakJaniels** [comment]
The loot message already supports them. Are you wanting a Center Screen Alert too?

**Reply by saenic** [reply]
Yeah I was thinking about a separate category besides the usual loot with CA and CSA.

**Reply by DakJaniels** [reply]
> Yeah I was thinking about a separate category besides the usual loot with CA and CSA.

![image](https://github.com/user-attachments/assets/6561898a-83f1-40e7-8334-2c96d5710e1f)

Better late than never I guess. Chat announcement for both in the dev branch. Still tweaking it.
